### DISCUSSION TITLE
Expose remaining parameters for labels in Spectrogram plugin

### DISCUSSION THREAD

**Discussion by thely** [katspaugh/wavesurfer.js#3820] [Ideas]
I'd love to see the ability to adjust the other parameters of the labels for the spectrogram plugin. I'd like to be able to easily adjust the font used and the alignment. I know this is doable because the options are available in the `loadLabels()` function that makes the labels, they're just not presented as options for the plugin itself. Perhaps `labels` could take a bool or an object, with the object encompassing all the other options, including font sizes and alignments?

[Here's a permalink to the function](https://github.com/katspaugh/wavesurfer.js/blob/a984cde75d10d2ef0d130118cc8f7c809d7d2243/src/plugins/spectrogram.ts#L637), relevant code is also below.

```js
// ...
private loadLabels(
    bgFill,
    fontSizeFreq,
    fontSizeUnit,
    fontType,
    textColorFreq,
    textColorUnit,
    textAlign,
    container,
    channels,
  ) {
    const frequenciesHeight = this.height
    bgFill = bgFill || 'rgba(68,68,68,0)'
    fontSizeFreq = fontSizeFreq || '12px'
    fontSizeUnit = fontSizeUnit || '12px'
    fontType = fontType || 'Helvetica'
    textColorFreq = textColorFreq || '#fff'
    textColorUnit = textColorUnit || '#fff'
    textAlign = textAlign || 'center'
    container = container || '#specLabels'
// ...
```

**Comment by katspaugh** [comment]
PRs are welcome!
